service/src/main/java/org/eclipse/keti/acs/service/policy/evaluation/PolicyEvaluationServiceImpl.java

```diff
@@ -249,29 +246,22 @@ private PolicyEvaluationResult handlePolicyEvaluationException(final PolicySet p
 
     boolean evaluateConditions(final Set<Attribute> subjectAttributes, final Set<Attribute> resourceAttributes,
             final String resourceURI, final List<Condition> conditions, final String resourceURITemplate) {
-        List<ConditionScript> validatedConditionScripts;
-        try {
-            validatedConditionScripts = this.policySetValidator.validatePolicyConditions(conditions);
-        } catch (PolicySetValidationException e) {
-            LOGGER.error("Unable to validate conditions: {}", e.getMessage());
-            throw new PolicyEvaluationException("Condition Validation failed", e);
-        }
-
         debugAttributes(subjectAttributes, resourceAttributes);
 
+        boolean result = true;
+
+        if ((conditions == null) || conditions.isEmpty()) {
```

Comment:
Replace conditions with:
```suggestion
        if (CollectionUtils.isEmpty(conditions)) {
```